article/app/controllers/ArticleController.scala

```diff
@@ -161,7 +161,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   def renderArticle(path: String) = {
     LongCacheAction { implicit request =>
-      mapModel(path) {
+      mapModel(path, request.isEmail) {
```

Comment:
Also why does email need to use `blocks`? Or more why can't we use `'blocks` everywhere?